### DISCUSSION TITLE
Getting hover_pos while dragging external file.

### DISCUSSION THREAD

**Discussion by Dapeos** [emilk/egui#946] [Q&A] [answered]
Thanks for the fantastic work. 
I am able to handle when an OS file is being dragged or dropped onto my egui interface. Works great. 

However I wanted to provide multiple drop buckets so when the user hovers with a file they can drop it in the appropriate bucket rather then a two step process. 

But hover_pos always is None, while dragging an external file. 

Just wondering if this is a known limitation or if I am doing something dumb.  

Thanks,

**Comment by emilk** [comment, marked as answer]
This is a limitation in `winit` (assuming you are on native). Feel free to open an issue on the winit repo!

**Comment by Dapeos** [comment]
Thanks emilk for responding, that makes sense. I ended up changing the the design for the better so didn't need the coordinates.
Thanks again,
